src/components/MessageActions/MessageActions.tsx

```diff
@@ -1,5 +1,7 @@
 import React, { useCallback, useEffect, useState } from 'react';
 
+import { ActionsIcon as DefaultActionsIcon } from '../Message/icons';
```

Comment:
gonna be super nitpicky and say move this to the spot right in front of isUserMuted lol